.editorconfig

```diff
@@ -42,3 +42,8 @@ end_of_line = lf
 
 [*.{razor,cshtml}]
 charset = utf-8-bom
+
+[*.cs]
+dotnet_diagnostic.CA1307.severity = error
```

Comment:
I think you want CA1310 here instead of CA1307 from the latest package.
CA1307 will continue to have its old behavior of being a readability/maintainability rule which provides the recommendation regardless of the default string comparison used by the flagged methid. It has been disabled by default as it can be noisy if you only care about correctness. CA1310 is the new rule which only flags string methods which are known to default to culture specific string comparisons, and is the correctness rule which will not be noisy.